packages/purpose-readmodel-writer/.env

```diff
@@ -0,0 +1,17 @@
+HOST=0.0.0.0
+PORT=3000
+LOG_LEVEL=info
+
+KAFKA_CLIENT_ID="purpose"
+KAFKA_GROUP_ID="purpose-group"
+KAFKA_BROKERS="localhost:9092"
+KAFKA_DISABLE_AWS_IAM_AUTH="true"
+PURPOSE_TOPIC="event-store.purpose.events"
+READMODEL_DB_HOST="localhost"
+READMODEL_DB_NAME="readmodel"
+READMODEL_DB_USERNAME="root"
+READMODEL_DB_PASSWORD="example"
+READMODEL_DB_PORT=27017
+AWS_REGION="eu-central-1"
+
+WELL_KNOWN_URLS="https://dev.interop.pagopa.it/.well-known/jwks.json"
```

Comment:
Same for this one, it is neede only for `*-process` services that receive HTTP requests and  validate a JWT (#424)

Reply:
Removed, thanks